### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:
-Use js in my form to check whether the email is in lowercase or uppercase and if  it is in uppercase, the form should be submitted 
else the user should be prompted to use a lowercase email

### CODE REVIEW

File: validate.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+function validateMail() {
+  const form = document.getElementById('myForm');
+  const email = form.elements.email.value;
```
Comment (ShoiraTa) [inline validate.js:3]: - [ ] If a user submits the contact form with the email field containing only lowercase characters the form should be sent normally, but it still shows an error after I correct the uppercase letter into a lowercase. 
Screenshot 2022-03-08 at 17 50 40

- [ ] OPTIONAL: I recommend you styling the error message and making it more user friendly. Currently it looks confusing, as you can see ✌🏻 
Screenshot 2022-03-08 at 17 52 41

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ksigei,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validate.js (hunk 1 above)
Comment (ksigei) [reply validate.js:3]: Hello, how do i solve this

**Review by AnselemOdims — APPROVED** [review]
### Hi  @ksigei 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**